docs/platform-qemu.md

```diff
@@ -21,13 +43,23 @@ supports a number of different disk formats.
 
 ## Networking
 
-TBD
+The `qemu` backend supports a number of networking options, depending
+on the platform you are running. The default is the userspace
+networking which provides the VM with a internal DHCP server and
+network connectivity, but does not provide access to the VMs network
+from the outside.
+
+With user mode networking you can publish selected VM ports on the
+host, using the `-publish` option. It uses the same syntax as the
+`qemu` binary.
+
+On Linux, you can attach the VM either to an existing bridge or tap interface.
 
-Also explain port forwarding
 
 ## Integration services and Metadata
 
-TBD: Integration services
+The `qemu` backend also allows passing custom userdata into the
+[metadata package](./metadata.md) using either the `-data` or `-data-file` command-line
+option. This attaches a CD device with the data on.
 
-The qemu backend currently does not support passing metadata/userdata
-to the VM.
+TBD: Integration services
```

Comment:
Does "Integration services" refer to the use of `pkg/qemu-ga`?

Reply:
yes